packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -29,102 +29,94 @@ export const styles = theme => ({
   },
 });
 
-class SpeedDialAction extends React.Component {
-  constructor(props) {
-    super();
-    this.state = {
-      tooltipOpen: props.tooltipOpen,
-    };
-  }
+const SpeedDialAction = ({
```

Comment:
Have you tried supporting ref forwarding?

Reply:
This was confusing for me since the prop injected by `withForwardedRef ` hoc `innerRef` was not being used anywhere in the code. So i didn't touch that part.  I converted it into `React.forwardRef` & assigned the `ref` to the root element of the component, was this the right approach to do it?